simple_trajectory_publisher:
  ros__parameters:
    use_sim_time: true
    moving_joint_names:
      - fer_joint1
      - fer_joint2
      - fer_joint3
      - fer_joint4
      - fer_joint5
      - fer_joint6
      - fer_joint7
    ee_frame_name: fer_hand_tcp
    w_q: [1.0]
    w_qdot: [1.0]
    w_qddot: [0.000001]
    w_robot_effort: [1.0]
    w_pose: [1.0]
    mask: [true, true, true, false, false, false]
    # mask: [True, True, True, True, True, True]
    trajectory_name: sine_wave_cartesian_space
    sine_wave:
      period: [4.0, 4.0, 0.0]
      scale_duration: [2.0, 2.0, 2.0]
      amplitude: [0.1, 0.1, 0.1]

# trajectory_name: sine_wave_configuration_space
# sine_wave:
#   period: [4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0]
#   scale_duration: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
#   amplitude: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
